### PR TITLE
Explain how to increase the log level if wanted

### PR DESCRIPTION
**Pull request by agoncal**

### CODE REVIEW

File: docs/content/workshop/01-slot1.md [hunk 1]
```diff
@@ -177,6 +177,14 @@ LLM can be slow to answer.
 60s is generally a good value. 
 However, feel free to adapt.
 
+???+ tip
```
Comment (cescoffier) [inline docs/content/workshop/01-slot1.md:182]: I would move the admonition tip after the "The last three properties configure the access to the Azure OpenAI service." sentence.

**Review by cescoffier — CHANGES_REQUESTED** [review]

File: docs/content/workshop/02-slot2.md [hunk 2]
```diff
@@ -103,6 +103,14 @@ The first property is used to specify the model to use.
 In our case, we will use the `llama2` model.
 The second property configures the timeout for the model.
 
+???+ tip
+If you want to visualise the request that is sent to the model and its response, you cam increase the log level by adding the following properties to the `src/main/resources/application.properties` file:
+
+    ```properties
+    quarkus.langchain4j.ollama.log-requests=true
```
Comment (cescoffier) [inline docs/content/workshop/02-slot2.md:110]: I fixed these levels a few days ago, You would need 0.7.0 to use them. However, it also changes the @AiService annotation.

Comment (agoncal) [reply docs/content/workshop/02-slot2.md:110]: So, shall I leave the tip then ?

Comment (cescoffier) [reply docs/content/workshop/02-slot2.md:110]: Yes, you are just living in the future :-)

**Review by cescoffier — APPROVED** [review]

**Pull request merged**